### ISSUE TITLE
Linebreak in toolbar

### ISSUE THREAD

**Issue by Falseee** [samclarke/SCEditor#731]
Hello,

is it possible to add a newline in the toolbar ?

So i got the following example Toolbar:
```
font,size,color|
bold,italic,underline|left,center,right
```
Instead of
```
font,size,color|italic,underline|left,center,right
```
?

**Comment by markmorris** [comment]
Yeah i also want to be able to add a BR tag from a button, but theres no where in the docs is there a clean example of how to do this, 10+ year developer and this is awkward to figure out.
